datacapturing/src/main/java/de/cyface/datacapturing/backend/DataCapturingBackgroundService.java

```diff
@@ -214,6 +223,12 @@ public int onStartCommand(final Intent intent, final int flags, final int startI
             final String authority = intent.getCharSequenceExtra(AUTHORITY_ID).toString();
             persistenceLayer = new MeasurementPersistence(this.getContentResolver(), authority);
 
+            // Loads EventHandlingStrategy
+            this.eventHandlingStrategy = intent.getParcelableExtra(EVENT_HANDLING_STRATEGY_ID);
+            if (eventHandlingStrategy == null) {
+                this.eventHandlingStrategy = new IgnoreEventsStrategy();
```

Comment:
Not sure if it's cleaner to allow library users to pass null if they don't care about those events or if we want to force them to pass the IgnoreEventsStrategy themselves if they don't want to implement a strategy themselves.

Reply:
It would be the cleanest if there are two constructors. One requiring the EventHandlingStrategy and another one using the default.